warehouse/organizations/models.py

```diff
@@ -189,13 +217,19 @@ class Organization(HasEvents, db.Model):
         nullable=True,
         onupdate=func.now(),
     )
+    customer_id = Column(
```

Comment:
Maybe not worth it in initial implementation since we _do_ plan to go with stripe, but should this be through another model? Like `StripeCustomers`?